department-of-veterans-affairs/va.gov-cms
# VACMS-5902: Set alert edit block forms to default draft.

## Description
Closes #5902

## Testing done
Visual

## QA steps
As an administrator, go to the edit form for a published alert block (block/2), and visually verify the moderation value is set to draft by default.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
